ui/src/app/shared/components/suspense-react-markdown-gfm.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Loading} from './loading';
```

Comment:
```suggestion

import {Loading} from './loading';
```

nit: split the external imports from the internal imports like I've done in other components (including `react-markdown-gfm`)